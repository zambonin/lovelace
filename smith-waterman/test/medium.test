#!/bin/sh

./smith-waterman/sw+ /dev/fd/3 3<< EOF1 /dev/fd/4 4<< EOF2
ACGTACGTACGT
EOF1
CTTGGACGTT
EOF2
