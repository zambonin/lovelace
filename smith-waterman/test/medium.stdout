Max Score is 10
